.chloggen/prwsubmit.yaml

```diff
@@ -0,0 +1,21 @@
+# Use this changelog template to create an entry for release notes.
+# If your change doesn't affect end users, such as a test fix or a tooling change,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
this should be a enhancement and not a bug fix.

Reply:
https://prometheus.io/docs/concepts/remote_write_spec/#ordering

> Prometheus Remote Write compatible senders MUST send samples for any given series in timestamp order.

Reply:
Isn't adhering to the spec not considered a bug fix?